src/main/antlr4/Manifold.g4

```diff
@@ -88,8 +89,14 @@ lvalue:
   reference # LValueExpression
   ;
 
+importExpr: 'import' STRING_VALUE;
+
 // TODO: declarations as expressions
-expression: /* declaration | */ rvalue;
+expression:
+    importExpr #ImportExpression
+  | rvalue #RValueExpression
+  // | declaration
+  ;
```

Comment:
Why do we need expression when it is only doing unit productions? Right now the semantics of expression are questionable. In what situations should we be treating import expressions and rvalue expressions interchangeably? For instance what is the meaning of using an import expression in a tuple value entry?